### ISSUE TITLE
Isolate View on Playblast

### ISSUE THREAD

**Issue by tokejepsen** [pypeclub/pype#367]
**Goal**

To be able to isolate the view on certain objects when playblasting.

**Comment by tokejepsen** [comment]
Just thinking aloud here, but there could be an option on the review instance to isolate the view to "animation" geometry only. That way you would only get the rigs and not the proxy geometry, and shader/material manipulation is not needed.

**Comment by tokejepsen** [comment]
I was envisioning an option called "Isolate View" where you could choose which family of instance to isolate the view to.

**Comment by tokejepsen** [comment]
@antirotor did you have some use cases for this as well?

**Comment by tokejepsen** [comment]
and @mkolar?

**Comment by mkolar** [comment]
Yes we have use for this. Mostly for modelling reviews actually. Clients often want to publish multiple LODs from a single maya scene, but to be able to review them individually, we'd need to trigger isolate view for each turnaround. It's actually not completely trivial in this case. Imagine a scene like this.

```
- model_high_GRP
- model_low_GRP
- model_proxy_GRP
- review_stage_GRP
```

The models are publishable instances, but the the stage is just helper geometry that we want to see in the review. For example a simple geometry platform model.

We'd need to be able to specify what exactly we want to isolate.

**Comment by tokejepsen** [comment]
So I guess for your use case @mkolar, we would need multiple review instances and somehow assign a single model instance to each?

The easiest workflow I can think of, is to be able to assign the any objectset to a review objectset. When publishing, if there are any objectsets in the review, only those will be shown, else everything in camera will be shown.
I dont really like the implicit nature of showing everything when no objectsets are assigned. Got any ideas to make this more explicit?
